Bottom: c032851dd1e3bb8cd6aba31545de9d6fc646f4cb
Top:    b7c94da2734922fcc6680e5574f94136ce7b5870
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 23:13:28 +0000

Refresh of play-with-quad-edge-data

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index e4f69c6..3b0351f 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -372,7 +372,7 @@ object3d_from_board_outline (void)
       /* Pick one of the upright edges which is within this face outer contour loop, and link it to the face */
       face3d_add_contour (faces[i], edges[2 * npoints + i]);
     }
-  faces[npoints]     = make_face3d (); /* bottom_face */
+  faces[npoints] = make_face3d (); /* bottom_face */
   faces[npoints + 1] = make_face3d (); /* top_face */
 
   /* Pick the first bottom / top edge which within the bottom / top face outer contour loop, and link it to the face */
